scalatest/src/main/scala/org/scalatest/tools/Fragment.scala

```diff
@@ -25,7 +25,7 @@ private[scalatest] case class Fragment(text: String, ansiColor: AnsiColor) {
     if (!presentInColor || text.trim.isEmpty) text
     else {
       ("\n" * countLeadingEOLs(text)) +
-      text.split("\n").dropWhile(_.isEmpty).map(ansiColor.code + _ + ansiReset).mkString("\n") +
+      text.split("\n").map(_.nn).dropWhile(_.isEmpty).map(ansiColor.code + _ + ansiReset).mkString("\n") +
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.